CharacterCreation/Class-Specific Documentation/Doctor Procedures.txt

```diff
@@ -101,6 +101,81 @@ payload of your choice.
     At level 10, has AP level 6 instead. At level 15, has AP level 8 and deals 
 65 damage. At level 20, ignores physical armor and deals 80 damage.
 
+== Poison: Pho-Ox ==
+
+Adrenaline Cost: 30 Points
+Contagion Cost: 1 Contagion
+Duration: 10 Rounds
+Range: Payload (use in combination with other Proceeedures)
+
+    This dichloroformaldoxime derivative is known as a "Nettle Agent" which 
+means that only extremely high doses will kill. Some might call that mercy, 
+while those it has been used on know better. Organics exposed to either the 
+liquid (which is a dull amber color with an oily texture) or the vapor almost
+immediately break out in hives, itch all over, serious eye irritation and 
+coughing fits. Extended or concentrated exposure can lead to patches of skin 
```

Comment:
Nonblocking 
Conjugation of effects does not match.
Suggest: break out in hives, itch all over, and suffer from serious...fits